static/src/components/AnnotationButton.vue

```diff
@@ -13,6 +13,8 @@ export default defineComponent({
      */
     label: { type: String, required: true },
     src: { type: String, required: true },
+    competency: { type: String, requred: true },
+    isAttentive: { type: Boolean, required: true },
```

Comment:
Hm, a little unfortunate that this actually rather "dumb" button now has to know such really not-at-all-related-to-it things. But I don't really have a better idea for now either